service/src/main/java/org/mskcc/smile/service/impl/TempoServiceImpl.java

```diff
@@ -41,11 +42,18 @@ public Tempo mergeBamCompleteEventBySamplePrimaryId(String primaryId, BamComplet
         return getDetailedTempoData(tempo);
     }
 
+    @Override
+    public Tempo mergeQcCompleteEventBySamplePrimaryId(String primaryId, QcComplete qcCompleteEvent) {
+        Tempo tempo = tempoRepository.mergeQcCompleteEventBySamplePrimaryId(primaryId, qcCompleteEvent);
+        return getDetailedTempoData(tempo);
+    }
+
```

Comment:
I noticed that there are two other QcComplete queries from `tempoRepository` that aren't being defined in this file: `findLatestQcCompleteEventByTempoId` and `findQcCompleteEventsByTempoId`. I assumed that's because we're not using them in the message handlers. If so, did we define these two queries so that we can use them when we need them in the future?

Reply:
Hard to know for sure if we'll use them since these are the early dev stages for TEMPO support but they're here just in case.